### PR TITLE
Test App for .NET 2.0 (Longevity) v1

### LINKED ISSUE #1208: Implement Mobile App for longevity test
- [x] Dockerized (.NET Standard Linux)
- [x] CRUD using 2.0 API

### PR DESCRIPTION
**Pull request by sethrosetter**
#### Fixes #1208.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:

- Add dotnet app for first pass of longevity testing
- Add dockerfile for dockerizing the app

### CODE REVIEW

**Comment by sethrosetter** [comment]
TODO
- [ ] Fix docker build
- [ ] Clean up deps
- [ ] Add Readme

File: apps/testkit.net/Testkit.Net.Core/Testkit.Net.Core.csproj.user [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'Default' ">
+    <StartAction>Project</StartAction>
+    <EnvironmentVariables>
+      <Variable name="LiteCoreLogSync" value="" xmlns="" />
```
Comment (raghusarangapani) [inline apps/testkit.net/Testkit.Net.Core/Testkit.Net.Core.csproj.user]: Do we want to set this to true?

File: docker/testkit-net/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Build the image
+```
+docker build -t testkit-net .
+```
+
+# Run the App
+
+--replication-endpoint is the endpoint <SyncGatewayUrl>:<DatabaseName> to run bidirectional replication against
```
Comment (raghusarangapani) [inline docker/testkit-net/README.md]: Based on the example below, it is SyncGatewayUrl / DatabaseName

File: apps/testkit.net/Testkit.Net.Core/Server.cs [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        internal string Test { get; set; }
+    }
+
+    public class Server
+    {
+        private readonly HttpListener _listener;
+
+        public Server(int port)
+        {
+            _listener = new HttpListener(IPAddress.Parse("127.0.0.1"), port);
```
Comment (sethrosetter) [inline apps/testkit.net/Testkit.Net.Core/Server.cs]: Correct. However, the REST server is a work in progress and is not being used at the moment. It should not block the PR in my opinion.

File: docker/testkit-net/README.md (hunk 2 above)
Comment (sethrosetter) [inline docker/testkit-net/README.md]: Done

File: apps/testkit.net/Testkit.Net.Core/Testkit.Net.Core.csproj.user (hunk 1 above)
Comment (sethrosetter) [inline apps/testkit.net/Testkit.Net.Core/Testkit.Net.Core.csproj.user]: No this works.

**Pull request merged**